src/main/scala/com/intersys/mdc/challenge/exercises/problems/Problem4.scala

```diff
@@ -55,15 +59,42 @@ case object Problem4 extends Problem {
 
           // A) Implicit class here with:
           // def asInt: Option[Int] = ???
-          ???
 
           // B) Implement the calculate method.
           // def calculate(ops: String, a: Int, b: Int): Option[Int] = ???
-          def calculate(ops: String, a: Int, b: Int): Option[Int] = ???
+          def calculate(ops: String, a: Int, b: Int): Option[Int] = {
+            ops match {
+              case "sum" => Some(a+b)
+              case "subtraction" => Some(a-b)
+              case "multiplication" => Some(a*b)
+              case "division" => try {
+                Some(a/b)
+              } catch {
+                case _: Exception => None
```

Comment:
👍 for considering the case when arithmetic exceptions (division by zero) might occur.